### ISSUE TITLE
being able to type a string value into a Asset Property 

### ISSUE THREAD

**Issue by TheBoxTrot** [Facepunch/sbox-issues#2906]
### For?

S&Box

### What can't you do?

![image](https://user-images.githubusercontent.com/29888245/218652082-573566a9-3e5e-44c6-80b2-c1df6701add1.png)
the ability to manually type in a string path to an Asset property such as a model. why would you want that just select it?. well what if you want to refer to a model that currently doesn't exist in your project. like when your making a addon for a gamemode and you know for a fact that this model exist in the gamemode you just type that path and when the addon is compiled with the gamemode it works. or with maps. since these properties are just fancy string paths anyway 

### How would you like it to work?

Right Clicking or double-clicking these type of properties will switch it to manual mode which will allow you to type in a file path to a model that currently does not exist in the project 

### What have you tried?

NA

**Comment by DoctorGurke** [comment]
I actually wanted to copy a model path from an asset similar to this the other day. Some way to do this easily would be lovely indeed

**Comment by TheBoxTrot** [comment]
yeah like the extra ui stuff is super useful most of the time but at the end of the day its just a string value so having the option to still set things with a string would be nice

**Comment by robotboy655** [comment]
![image](https://user-images.githubusercontent.com/3299036/218761479-28367aed-97e1-4ccc-bc98-e472e917bc9a.png)

**Comment by TheBoxTrot** [comment]
thats nice to have but that wasnt what I was talking about in my issue. i was saying being able to instead of selecting a sound or a model or material or whatever we had the option to also just manually type in a path as we currently have no way to refer to those resources if their not in our current project

**Comment by robotboy655** [comment]
If it's not in your project, how do you expect it to work?

**Comment by TheBoxTrot** [comment]
say for examples addons for gamemodes i don't currently have the gamemode models but if I type the path to a model I know exist in the gamemode when the game compiles it will see the model and it all works

**Comment by robotboy655** [comment]
If you are developing an addon for a gamemode, surely you'd have the gamemode installed for testing and such?

**Comment by TheBoxTrot** [comment]
sometimes not, for example, my tardis gamemode the addons are just gameresources that's it. so to download the entire gamemode just to refer to a model is a bit excessive especially since you don't need the gamemode local to test it cause the live version will load the addons files

**Comment by robotboy655** [comment]
I'd like to see some more opinions on whether that's something other people want/need before looking into adding that.

**Comment by TheBoxTrot** [comment]
yea that fine. my opinion is just that the property is a string anyway so being able to treat it as a string if you want would be a good extra option
